stdlib/typing_extensions.pyi

```diff
@@ -62,7 +62,7 @@ class _TypedDict(Mapping[str, object], metaclass=abc.ABCMeta):
     __required_keys__: frozenset[str]
     __optional_keys__: frozenset[str]
     __total__: bool
-    def copy(self: _T) -> _T: ...
+    def copy(self: _T) -> _T: ...  # noqa: Y019
```

Comment:
Why can't this use Self?

Reply:
Because of line 45 — `typing_extensions.Self` is a `_SpecialForm`, not a `TypeVar`. I deliberately excluded `typing_extensions` from my script for that reason.

Reply:
Maybe use `from _typeshed import Self as _Self`?

Reply:
We'd still need a `# noqa` comment, though, because the id of the `ast.Name` node would be `_Self` rather than `Self`

Reply:
We could do `from _typeshed import Self as TypeshedSelf` and it would pass without a `# noqa` — my proposed Y019 only raises errors for names starting with a leading underscore.

Reply:
Let's do that, that will also make it obvious that we can switch to PEP 673 Self when that day comes.